### PR TITLE
[stable/kube-state-metrics] added resource config to values.yaml

### PR DESCRIPTION
**Pull request by monotek**
Signed-off-by: André Bauer <monotek23@gmail.com>

#### What this PR does / why we need it:
* added resource config example in the values as its mentioned in the readme

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by fiunchinho** [comment]
@tariq1890 It seems people can't assign me. Can you remove me as maintainer? Thanks

**Comment by monotek** [comment]
Imho you can still /lgtm ;-)

File: stable/kube-state-metrics/values.yaml [hunk 1]
```diff
@@ -133,7 +133,20 @@ collectors:
...
+  #  memory: 64Mi
+  # requests:
+  #  cpu: 10m
+  #  memory: 32Mi
+
 ## Provide a k8s version to define apiGroups for podSecurityPolicy Cluster Role.
 ## For example: kubeTargetVersionOverride: 1.14.9
 ##
 kubeTargetVersionOverride: ""
+
```
Comment (mrueg) [inline stable/kube-state-metrics/values.yaml]: can you remove the empty line here?

Comment (monotek) [reply stable/kube-state-metrics/values.yaml]: done :)

**Review by tariq1890 — APPROVED** [review]
/lgtm

File: stable/kube-state-metrics/Chart.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ keywords:
 - monitoring
 - prometheus
 - kubernetes
-version: 2.8.11
+version: 2.8.12
```
Comment (tariq1890) [inline stable/kube-state-metrics/Chart.yaml]:
```suggestion
version: 2.8.13
```

Comment (tariq1890) [reply stable/kube-state-metrics/Chart.yaml]: can you rebase your PR?

**Pull request merged**